### PR TITLE
Update free-programming-interactive-tutorials-en.md

### PR DESCRIPTION
**Pull request by mawad28**
## What does this PR do?
Add resource

### Description

Add Laravel Bootcamp, official guide from Laravel team.

### Why is this valuable (or not)?

This guide help developers that are new to the framework get up and going quickly.

### How do we know it's really free?

It's Laravel official site.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -1786,6 +1786,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 
 * [100 (and counting) Laravel Quick Tips](https://laraveldaily.com/wp-content/uploads/2020/04/laravel-tips-2020-04.pdf) - Povilas Korop / LaravelDaily Team (PDF)
 * [Laravel Best Practices](http://www.laravelbestpractices.com)
+* [Laravel Bootcamp](http://bootcamp.laravel.com) - Laravel Team (HTML)
```
Comment (shrn01) [inline books/free-programming-books-langs.md:1789]: Looks like this is a highlevel guide which does not give more information about the framework as such. Is this a value add?

Comment (mawad28) [reply books/free-programming-books-langs.md:1789]: I think this guide is more about project hands-on (building a microblogging platform) for more detailed we can see the documentation, there are 2 paths in this guide:
- using blade (beginner friendly)
- using modern js frontend stack (vue and react though inertia)

Comment (eshellman) [reply books/free-programming-books-langs.md:1789]: Please move this to interactive tutorials

Comment (mawad28) [reply books/free-programming-books-langs.md]: moved item to interactive tutorials

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

File: more/free-programming-interactive-tutorials-en.md [hunk 2]
```diff
@@ -335,6 +336,11 @@
 * [PHP tutorial](https://www.w3schools.com/php) - W3Schools
 
 
+#### Laravel
+
+* [Laravel Bootcamp](http://bootcamp.laravel.com) - Laravel Team (HTML)
```
Comment (eshellman) [inline more/free-programming-interactive-tutorials-en.md:341]:
```suggestion
* [PHP Framework for Web Artisans](http://bootcamp.laravel.com) - Laravel Team
```

**Pull request merged**